low-level/sensors/impl/centos_7/systemd_watchdog.py

```diff
@@ -977,10 +978,10 @@ def _interface_removed(self, object_path, interfaces):
                 if interface == "org.freedesktop.UDisks2.Drive":
                     with self._drive_info_lock:
                         # Speed thread up in case we have multiple drive removal events queued up
-                        self._thread_sleep = .10

```

Comment:
add comments on why you are commenting. for future references. same for line 981, 984

Reply:
done